schoolopy/main.py

```diff
@@ -56,7 +60,7 @@ def _post(self, path, data):
         :return: JSON response.
         """
         try:
-            return requests.post(self._ROOT + path, json=data, headers=self._request_header()).json()
+            return self.schoology_auth.oauth.post(url='%s%s?limit=%s' % (self._ROOT, path, self.limit), json=data, headers=self._request_header()).json()
```

Comment:
I think I made the limit change upstream as well, you'll want to pull my updates.